src/main/java/frc/robot/CommandSwerveDrivetrain.java

```diff
@@ -55,6 +68,42 @@ private void startSimThread() {
     m_simNotifier.startPeriodic(kSimLoopPeriod);
   }
 
+  public SwerveRequest drive(double x, double y, double r){
```

Comment:
The method is in accurate. This function isn't telling the drive train to move, it is getting the request to submit to tell the drive train to move. something along the the lines of `getDriveRequest(...)` would be a better name

Also comment stating the expected input range would be useful, and changing r to a different name.
r could be thought as rotation but that isn't what it represents. With it representing angular velocity it would be closer to alpha then r.